src/pylutron_caseta/leap.py

```diff
@@ -1,7 +1,7 @@
 """LEAP protocol layer."""
 
 import asyncio
-import json
+import orjson as json
```

Comment:
My preference would be to keep this imported as `orjson` instead of `json` since they have slightly different APIs and it can be confusing if you are working below and don't scroll up to realize its not standard lib `json`